docs/pre-commit-hooks.md

```diff
@@ -58,7 +48,7 @@ This hook is provided by the [Export Poetry Plugin](https://github.com/python-po
 {{% /warning %}}
 
 {{% note %}}
-It is recommended to run the [`poetry-lock`](#poetry-lock) hook prior to this one.
```

Comment:
IMO that's the only part that might be changed. Instead of recommending only `poetry-lock`, we may recommend two alternatives: `poetry-lock` or `poetry-check` with `--lock`.

Reply:
gotcha, will update the same.